diesel_cli/Cargo.toml

```diff
@@ -27,7 +27,7 @@ regex = "0.1.48"
 
 [features]
 default = ["postgres", "sqlite", "mysql"]
-lint = ["clippy"]
+lint = ["clippy", "diesel/lint", "diesel_infer_schema/lint"]
```

Comment:
Whoops, I didn't mean to add this here. I was trying to figure out why `bin/test` twice no longer runs the second time with 0 recompilations and I think it's because crates are recompiled with/without clippy, but the doctest dependency on `diesel_codegen` in `diesel` breaks everything (as usual. What else is new)